modules/ROOT/pages/design-import-files-api-fragment-projects.adoc

```diff
@@ -0,0 +1,58 @@
+= Import Files into an API-Fragment Project
+
+You can import RAML fragments into an API-fragment project. The fragments that you import can be written in RAML 0.8 or RAML 1.0. You can import annotation types, data types, examples of JSON response payloads, libraries, resource types, security schemes, and traits. You must set one of the fragments in your project as the root file of the project.
+
+There are three locations that you can import from: your computer, a location at a URL, and Anypoint Exchange.
+
+== Procedure
+
+* Import a file or the content of a `.zip` file that is stored on your computer, if you are using the text editor:
+. Click the gear icon in the top-right corner of the text editor and select *Import*.
```

Comment:
```suggestion
. Click the gear icon in the text editor and select *Import*.
```

Reply:
@kbostrom , the text editor fills an entire browser window. If I don't explain where the icon is, users will not find it easily.